File: src/my_project_typescript/easy_problems_typescript/from51To100/minimumNumberMovesSeatEveryone.ts
function minMovesToSeat(seats: number[], students: number[]): number {

    let answer: number = 0
    seats.sort((x,y) => x<=y?-1:1)
    students.sort((x,y) => x<=y?-1:1)
    let lenSeats: number = seats.length

    for (let i = 0; i < lenSeats; i++){
        answer += Math.abs(seats[i] - students[i])
    }

    return answer 
};


console.log(minMovesToSeat([3,1,5],  [2,7,4]))

console.log(minMovesToSeat([4,1,5,9],  [1,3,2,6]))

console.log(minMovesToSeat([2,2,6,6],  [1,3,2,6]))


// funciton to be removed

function newFunction(num: number){
    return num*7
}

File: src/my_project/easy_problems/from151To200/minimum_number_moves_seat_everyone.py
from typing import List 

class Solution:
    def minMovesToSeat(self, seats: List[int], students: List[int]) -> int:

        answer: int = 0
        seats.sort()
        students.sort()
        len_seats: int = len(seats)

        for i in range(len_seats):
            answer += abs(seats[i] - students[i])

        return answer 


solution = Solution()

print(solution.minMovesToSeat(seats = [3,1,5], students = [2,7,4]))

print(solution.minMovesToSeat(seats = [4,1,5,9], students = [1,3,2,6]))

print(solution.minMovesToSeat(seats = [2,2,6,6], students = [1,3,2,6]))
